src/tool/subcommands/api_cmd/test_snapshots.txt

```diff
@@ -18,7 +18,9 @@ filecoin_chaintipsetweight_1736937942819105.rpcsnap.json.zst
 filecoin_ethaccounts_1737446676689764.rpcsnap.json.zst
 filecoin_ethaddresstofilecoinaddress_1740132538465536.rpcsnap.json.zst
 filecoin_ethblocknumber_1741272348346171.rpcsnap.json.zst
-filecoin_ethcall_1737446676693468.rpcsnap.json.zst
+filecoin_ethcall_1744204533050503.rpcsnap.json.zst
+filecoin_ethcall_1744204533058637.rpcsnap.json.zst
+filecoin_ethcall_1744204533066529.rpcsnap.json.zst
```

Comment:
What are the differences between each of those snapshots?

Reply:
Each one contain different `eth_calls`:
 1. No data and `to` field set to `0x0000000000000000000000000000000000000000`
 2. Contains a token transfer call to an `0x0c1d86d34e469770339b53613f3a2343accd62cb`
 3. Contract deployment call
 
 Generated from these [cases](https://github.com/ChainSafe/forest/blob/56da0ea7bb0de5a26300154113fcda71501c50b5/src/tool/subcommands/api_cmd/api_compare_tests.rs#L1208).

Reply:
As @akaladarshi mentioned, it's generated from the same API compare calls as before, just re-exported to capture the new usage of the chainstore.